e2e/suites/management/create_new_scheduler_version_test.go

```diff
@@ -181,6 +183,53 @@ func TestCreateNewSchedulerVersion(t *testing.T) {
 			require.NoError(t, err)
 			require.NotNil(t, updateResponse.OperationId, scheduler.Name)
 
+			// Wait till we have 3 pods (the third one is the validation one). Then, try to forward event.
+			// Since we don't forward event from validation room, it should return true even though we're not
+			// mocking the gRPC call.
+			require.Eventually(t, func() bool {
```

Comment:
@caiordjesus i just changed the index of the pod we test to 1 (second pod) and the test is still passing... With further investigation I noticed that the scheduler we are creating in this test does not have the forwarder configured, this test will always pass. We need to add the forwarder.